cmd/suggestion/nas/enas/v1beta1/requirements.txt

```diff
@@ -1,5 +1,6 @@
 grpcio==1.41.1
 protobuf==3.19.1
 googleapis-common-protos==1.6.0
-tensorflow==2.8.0
+tensorflow==2.9.1; platform_machine=="x86_64"
+tensorflow-aarch64==2.9.0; platform_machine=="aarch64"
```

Comment:
It looks better condition at requirements.txt :+1: 

nit question: don't we have to match the hotfix version?

Reply:
Thanks for your comment.
The tensorflow-aarch64  v2.9.1 has not yet been released.
Ref: https://pypi.org/project/tensorflow-aarch64/

Reply:
Then does it better to set both to v2.9.0 until aarch64 support v2.9.1?

Reply:
It seems that the Tensorflow v2.9.0 has installation issues. So it would be best to wait for upgrading until the released tensorflow-aarch64 v2.9.1. Thanks for your suggestion @anencore94 !

Ref: https://github.com/tensorflow/tensorflow/issues/56077

Reply:
Ah, I didn't know that! Thanks for the clarifying :)

Reply:
We decided to merge this PR without waiting for tensorflow-aarch64 v2.9.1 as discussed in https://github.com/kubeflow/katib/pull/1882#discussion_r889194123